src/elementary/test/elementary.test.cpp

```diff
@@ -128,3 +126,142 @@ SCENARIO( "split" ) {
     } // THEN
   } // GIVEN
 } // SCENARIO
+
+SCENARIO( "toEndfReactionNumber" ) {
+
+  GIVEN( "a valid reaction identifier compatible with ENDF" ) {
+
+    THEN( "the ENDF MT number can be retrieved" ) {
+
+      // incident neutrons
+      CHECK( 2 == toEndfReactionNumber( ReactionID( "n,Fe56->n,Fe56" ) ) );

```

Comment:
Would MT=1 be `n,Fe56->total`? Would fission be `n,U235->fission`?

Reply:
Yes, but fission is a bit trickier since there would be a reaction qualifier:
- fission[total] for total fission
- fission for first chance fission
- fission[n] for fission after emission of a neutron, aka first second fission
- fission[2n] for third chance
I had a discussion with Caleb, doing it this way would be better if an evaluator would like to specify more generic fission reactions, for instance fission[n,p] for a third chance fission reaction emitting a neutron and proton before fission.

Fission would be the only "fundamental" reaction without a particle based name. Summation or derived reactions would be a descriptive name. E.g. inelastic for MT4, absorption, total, etc.

For level summations, we could consider the following as well: n,Fe56->n,Fe56[total] but it might be confusing with elastic. For n,2n summation it would work though: n,Fe56->2n,Fe55[total] for MT16 and n,Fe56->2n,Fe55 for the reaction to the ground state of Fe55.

A similar qualifier would be applied for the continuum reactions (e.g. MT91) but allow for more flexibility, e.g. n,Fe56->n,Fe56[e10-e15] or something similar to specify a continuum that encompasses excited levels 10 to 15.